content/hardware/09.kits/maker/nesso-n1/tutorials/user-manual/content.md

```diff
@@ -72,25 +71,53 @@ The full datasheet is available as a downloadable PDF from the link below:
 
 ## Installation
 
-The Nesso N1 can be programmed using the Arduino IDE or the Arduino Cloud Editor. To get started, you will need to install the appropriate board package.
+The Nesso N1 is programmed using the desktop Arduino IDE. To get started, you will need to install the appropriate board package.
 
 ### Arduino IDE
 
-To use the board in the Arduino IDE, you need to install the latest version of the **esp32 by Espressif Systems** package from the boards manager.
+To use the board in the Arduino IDE, you must install the latest version of the **esp32 by Espressif Systems** package. Support for the Nesso N1 requires version **3.3.3** or newer.
 
 1.  Open the Arduino IDE.
 2.  Navigate to **Boards Manager** (**Tools > Board > Boards Manager...**).
 3.  Search for **"esp32"** and find the package by **Espressif Systems**.
-4.  Click the **Install** button.
+4.  Click the **Install** (or Update) button.
 5.  Once installed, select **Arduino Nesso N1** from the **Tools > Board > esp32** menu.
 
 ![Installing the esp32 Boards core in the Arduino IDE](assets/board-manager.png)
 
 ### Arduino Cloud Editor
 
-The Arduino Cloud Editor is an online IDE that supports the Nesso N1 without requiring manual installation of the board package.
+Direct support for the Nesso N1 in the **Arduino Cloud Editor** (the online web IDE) is coming soon. Currently, the Cloud Editor does not support the specific ESP32 core version required for this board.
 
-Read more in the [Getting Started with the Cloud Editor](https://docs.arduino.cc/arduino-cloud/guides/editor/) guide.
+Please use the **Arduino IDE** (desktop version) to compile and upload code to the Nesso N1.
+
+## Arduino IoT Cloud
+
+Although the Nesso N1 cannot yet be programmed directly via the Cloud Editor, you can still use it with **Arduino IoT Cloud** dashboards and variables by configuring it as a "Manual Device" and uploading the sketch from your desktop IDE.
+
+### 1. Create a Manual Device
+1.  Go to the [Arduino IoT Cloud Devices page](https://app.arduino.cc/devices).
+2.  Click **+ CREATE**.
+3.  Select **Any Device** (do not search for Nesso N1).
+4.  Click **Continue**.
+5.  Name your device (e.g., "MyNessoN1") and click **Next**.
+6.  **Important:** Save the **Device ID** and **Secret Key**. You will need these later in your code.
+7.  Check the box confirming you have saved your credentials and click **Continue**.
+
+### 2. Create a Thing
+1.  Go to the [Things page](https://app.arduino.cc/things).
+2.  Create a new Thing and associate it with the "Manual Device" you just created.
+3.  Add your desired Cloud Variables (e.g., an LED switch or sensor value).
+
+### 3. Program the Board via Desktop IDE
+1.  Open the **Arduino IDE** on your computer.
+2.  Install the **ArduinoIoTCloud** library via the Library Manager.
+3.  Copy the sketch generated by the Cloud "Thing" (from the "Sketch" tab in the web interface) into the Arduino IDE.
+4.  Open the `thingProperties.h` tab in your local sketch.
```

Comment:
It is true that the credentials can be configured via the `thingProperties.h` tab. However, the established convention is to do this via `arduino_secrets.h` ([example](https://github.com/arduino-libraries/ArduinoIoTCloud/blob/master/examples/ArduinoIoTCloud-Basic/arduino_secrets.h)). If the user is starting from a sketch that has `arduino_secrets.h`, I think it would be confusing for them to instead configure credentials via `thingProperties.h`.

The exception is that the convention is to define the Device ID via `thingProperties.h` (since this information is not a secret).